frontend/src/app/AboutDialog.tsx

```diff
@@ -81,6 +120,62 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
                 : 'Unknown'}
             </Content>
           </Content>
+
+          {/* Component Releases Section */}
+          <Content
+            style={{
+              paddingTop: 48,
+              visibility: loading ? 'hidden' : 'visible',
+            }}
+          >
+            <h4>Installed components</h4>
+            {groupedComponents.length > 0 ? (
+              <Table aria-label="Component Releases Table" data-testid="component-releases-table">
+                <Thead>
+                  <Tr data-testid="table-row-title">
+                    <Th modifier="wrap">
+                      {isRHOAI ? RhoaiDefaultComponentReleaseName : OdhDefaultComponentReleaseName}{' '}
+                      Component
+                    </Th>
+                    <Th modifier="wrap">Upstream component</Th>
+                    <Th modifier="wrap">Upstream version</Th>
+                  </Tr>
+                </Thead>
+                <Tbody>
+                  {groupedComponents.map(([displayName, details]) =>
+                    details?.releases && details.releases.length > 0 ? (
+                      details.releases.map((release, index) => (
+                        <Tr key={`${displayName}-${index}`} data-testid="table-row-data">
+                          {index === 0 ? (
+                            <Td rowSpan={details.releases.length}>{displayName}</Td>
+                          ) : null}
+                          <Td
+                            style={{
+                              paddingInlineStart: 'var(--pf-v6-c-table--cell--Padding--base)',
+                            }}
+                          >
+                            {release.repoUrl ? (
+                              <ExternalLink text={release.name} to={release.repoUrl} />
+                            ) : (
+                              release.name
+                            )}
+                          </Td>
+                          <Td>{release.version}</Td>
+                        </Tr>
+                      ))
+                    ) : (
+                      <Tr key={displayName}>
+                        <Td>{displayName}</Td>
+                        <Td colSpan={3}>No releases available</Td>
+                      </Tr>
+                    ),
+                  )}
+                </Tbody>
+              </Table>
+            ) : (
+              <p>No component releases available</p>
```

Comment:
The title is installed components, so the empty state may as well relate to it:
```suggestion
              <p>No installed components</p>
```